ScienceJournal/UI/MaterialHeaderViewController.swift

```diff
@@ -28,7 +28,8 @@ class MaterialHeaderViewController: ScienceJournalViewController, UIScrollViewDe
 
   let appBar = MDCAppBar()
   var trackedScrollView: UIScrollView? { return nil }
-  @objc dynamic private(set) var isContentOutsideOfSafeArea: Bool = false
+  @objc private(set) lazy var scrollViewContentObserver
+    = ScrollViewContentObserver(scrollView: trackedScrollView)
```

Comment:
Nit: Formatting-wise in SJ we do the `=` on the preceding line instead, like:

```
@objc private(set) lazy var scrollViewContentObserver =
  ScrollViewContentObserver(scrollView: trackedScrollView)
```

Reply:
Ah, thanks; done.